robottelo/hosts.py

```diff
@@ -1374,3 +1374,20 @@ def register_to_dogfood(self):
             raise SatelliteHostError(
                 f'Error during registration, command output: {cmd_result.stdout}'
             )
+
+    def install_cockpit(self):
+        cmd_result = self.execute(
+            'foreman-installer --enable-foreman-plugin-remote-execution-cockpit', timeout='30m'
+        )
+        if cmd_result.status != 0:
+            raise SatelliteHostError(
+                f'Error during cockpit installation, installation output: {cmd_result.stdout}'
+            )
+        cmd_result = self.execute(
+            f'sshpass -p "{settings.server.ssh_password}" ssh-copy-id \
+            -i ~foreman-proxy/.ssh/id_rsa_foreman_proxy -o StrictHostKeyChecking=no localhost'
```

Comment:
IIRC this is key used for REX? I wonder if you could use:
`robottelo]$ grep -r add_rex_key robottelo/
robottelo/hosts.py:    def add_rex_key(self, satellite, key_path=None)`

Reply:
will check this one

Reply:
@swadeley @tstrych This could be a non-blocking comment though as he just moved the existing code to new place and later he might raise another PR as improvement !